packages/airnode-node/src/api/index.ts

```diff
@@ -235,10 +235,11 @@ export async function processSuccessfulApiCall(
   }
 
   const goExtractAndEncodeResponse = goSync(() =>
-    adapter.extractAndEncodeResponse(
-      goPostProcessApiSpecifications.data,
-      reservedParameters as adapter.ReservedParameters
-    )
+    adapter.extractAndEncodeResponse(goPostProcessApiSpecifications.data, {
+      _type,
+      _path,
+      _times,
+    } as adapter.ResponseReservedParameters)
```

Comment:
Similarly here, trying to be explicit about which reserved parameters are relevant to response processing.